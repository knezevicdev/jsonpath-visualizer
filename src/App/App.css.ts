import styled from 'styled-components';

export const Wrapper = styled.h1`
  color: red;
`;
