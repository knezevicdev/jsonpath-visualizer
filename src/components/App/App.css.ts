import styled from 'styled-components';

export const TopWrapper = styled.div`
  height: 10vh;
  display: flex;
  align-items: center;
  justify-content: space-evenly;
`;
