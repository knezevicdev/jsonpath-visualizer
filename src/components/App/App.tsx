import React from 'react';
import { ToastProvider } from 'react-toast-notifications';

import FileSelector from '../FileSelector/FileSelector';
import JSONView from '../JSONView/JSONView';
import { withStore } from 'utils/store';

const App: React.FC = () => {
  return (
    <ToastProvider autoDismiss>
      <FileSelector />
      <JSONView />
    </ToastProvider>
  );
};

export default withStore(App);
