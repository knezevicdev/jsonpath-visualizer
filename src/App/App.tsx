import React from 'react';

import { Wrapper } from './App.css';

const App: React.FC = () => {
  return <Wrapper>Test</Wrapper>;
};

export default App;
