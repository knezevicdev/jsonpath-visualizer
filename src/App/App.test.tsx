import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('App', () => {
  it('renders', () => {
    const wrapper = shallow(<App />);
    expect(wrapper.html()).toMatch(/Test/);
  });

  it('renders snapshots', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
