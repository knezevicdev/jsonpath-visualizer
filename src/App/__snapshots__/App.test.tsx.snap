// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders snapshots 1`] = `
<styled.h1>
  Test
</styled.h1>
`;
